### DISCUSSION TITLE
The number of entries per feed is sometimes to small

### DISCUSSION THREAD

**Discussion by SmutnyStefan** [tughi/aggregator-android#56] [Ideas]
I don't always read actual news per se, but rather I follow serialized blogs, that ought to be read in sequential order. Therefore it ofen happens that I neeed to go back more than two or three entries, especially when I join a new blog, or when I forget to check one for a longer time. Could this be somehow solved for example by allowing to specify the number of articles per feed, or by increasing the general number of available entries in a feed?

**Comment by tughi** [comment]
Aggregator uses the configured "cleanup mode" to delete old feed entries. The default cleanup mode is set to one month. You might want to change that...

**Comment by SmutnyStefan** [comment]
Yes, thanks, this resolved the problem. However it is still buggy, sort of. To increase the numer of available past entries first  I had to remove, and then re-ad the feed with cleanup level increased. When i was simply increasing the cleanup threshold and then clicking update it had no effect. That is the reason why I opened the issue, since I thought it does not work at all

**Reply by tughi** [reply]
That's a known issue... The old entries will be aggregated only when the feed gets updated.

This can be improved by resetting the cached etag and last-modification values...
